backend/tests/unit/core/test_attribute.py

```diff
@@ -99,6 +99,40 @@ async def test_validate_validate_url(db: InfrahubDatabase, default_branch: Branc
     )
 
 
+async def test_validate_format_datetime_valid(
+    db: InfrahubDatabase, default_branch: Branch, criticality_schema: NodeSchema
+):
+    schema = criticality_schema.get_attribute("time")
+
+    assert DateTime(
+        name="test", schema=schema, branch=default_branch, at=Timestamp(), node=None, data=current_timestamp()
+    )
+    assert DateTime(
+        name="test", schema=schema, branch=default_branch, at=Timestamp(), node=None, data="2024-06-04T03:13:03.386270"
+    )
+
+
+async def test_validate_format_datetime_invalid(
+    db: InfrahubDatabase, default_branch: Branch, criticality_schema: NodeSchema
+):
+    schema = criticality_schema.get_attribute("time")
+
+    with pytest.raises(ValidationError) as error:
+        DateTime(name="test", schema=schema, branch=default_branch, at=Timestamp(), node=None, data="invalid-datetime")
+    assert error.value.args[0] == "invalid-datetime is not a valid DateTime"
```

Comment:
you might be able to move this into a `match` kwarg in `pytest.raises`
something like 
```python
with pytest.raises(ValidationError, match=r"invalid-datetime is not a valid DateTime"):
    ...
```

Reply:
I'm pretty sure this was on of your feedbacks previously lol Thank you!